ios/imagemounter/personalized_image_mounter.go

```diff
@@ -0,0 +1,196 @@
+package imagemounter
+
+import (
+	"fmt"
+	"github.com/Masterminds/semver"
+	"github.com/danielpaulus/go-ios/ios"
+	log "github.com/sirupsen/logrus"
+	"io"
+	"os"
+	"path"
+)
+
+type PersonalizedDeveloperDiskImageMounter struct {
+	deviceConn ios.DeviceConnectionInterface
+	plistRw    ios.PlistCodecReadWriter
+	version    *semver.Version
+	tss        tssClient
+	ecid       uint64
+}
+
+func NewPersonalizedDeveloperDiskImageMounter(entry ios.DeviceEntry, version *semver.Version) (PersonalizedDeveloperDiskImageMounter, error) {
+	values, err := ios.GetValuesPlist(entry)
+	if err != nil {
+		return PersonalizedDeveloperDiskImageMounter{}, nil
```

Comment:
@dmissmann you are swallowing the error here, please wrap it like fmt.Errorf("NewPersonalizedDeveloperDiskImageMounter: failed to get lockdown values from device with error: %w", err)